docs/content/ecs/world-commands.md

```diff
@@ -0,0 +1,126 @@
+**Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+
+## ECS: World commands
+
+World commands are special commands that are sent to the SpatialOS runtime to ask it to reserve entity ids, create or delete entities, or request information about entities. (See the SpatialOS documentation on [world commands](https://docs.improbable.io/reference/latest/shared/design/commands#world-commands) for more information.) 
+
+Each ECS entity that represents a SpatialOS entity has a set of components for sending world commands. For each world command, there is a component to send the command and receive the response. 
+
+### 1. Reserve an entity ID
+
+You can use the `ReserveEntityIds` world command before entity creation which makes it easier to create multiple entities in a group.
```

Comment:
Would reword this to:

> You can use the `ReserveEntityIds` world command to reserve groups of entity IDs that you can use in entity creation.

Or something similar - it doesn't really make it easier to create entities in a group.